src/util/manifest.js

```diff
@@ -45,8 +46,9 @@ export default async function getValidatedManifest(
   let manifestData;
 
   try {
-    manifestData = JSON.parse(manifestContents);
+    manifestData = parseJSON(manifestContents, manifestFile);
   } catch (error) {
+    error.fileName = manifestFile;
     throw new InvalidManifest(
       `Error parsing manifest.json at ${manifestFile}: ${error}`);
```

Comment:
we can remove the `at ${manifestFile}` part here, because the full path name is now part of the error raised by parseJSON